operations/sftpgo-authentication-service/internal/httpclient/client.go

```
@@ -0,0 +1,130 @@
+func (t *LoggingTransport) logRequest(req *http.Request) {
+	t.Logger.Debug("OUTGOING REQUEST: method=%s url=%s", req.Method, req.URL.String())
+
+	if t.Logger.IsTraceEnabled() {
+		t.Logger.Trace("OUTGOING REQUEST HEADERS: %v", sanitizeHeaders(req.Header))
+
+		if req.Body != nil {
+			const logLimit = 10240
+			prefix := make([]byte, logLimit)
+			n, err := io.ReadFull(req.Body, prefix)
+
+			switch {
```

golangci-lint (2.12.2)

[error] QF1002: could use tagged switch on err

(staticcheck)